Defs/Ammo/AmmoCategoryDefs/AmmoCategories_Advanced.xml

```diff
@@ -69,4 +69,40 @@
     <description>Contains ionising gamma radiation that significantly impairs consciousness, and is a carcinogen.</description>
   </CombatExtended.AmmoCategoryDef>
   
+  <!-- ========== For Project Red Horse: Militaires Sans Frontieres ========== -->
+  
+  <CombatExtended.AmmoCategoryDef>
```

Comment:
The tranq ammo category should probably into AmmoCategories_General since it's an ammo type for conventional rifle/pistol calibers.

Reply:
Fixed in edc2a6f